Dockerfile

```diff
@@ -5,41 +5,59 @@ FROM base AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
-COPY package.json package-lock.json ./
+
+# Install dependencies based on the preferred package manager
+COPY package.json package-lock.json*  ./
```

Comment:
@G3root Why the * here - `package-lock.json*`?

Reply:
i got it from nextjs example repo. sent a fix to remove that.